modules/storage-rocksdb/src/test/java/org/apache/ignite/internal/storage/rocksdb/RocksDbMvPartitionStorageConcurrencyTest.java

```diff
@@ -49,7 +49,7 @@ void setUp(
             @WorkDirectory Path workDir,
             @InjectConfiguration("mock.flushDelayMillis = 0")
             RocksDbStorageEngineConfiguration engineConfig,
-            @InjectConfiguration("mock.profiles.default = {engine = rocksdb, size = 16777216, writeBufferSize = 16777216}")
+            @InjectConfiguration("mock.profiles.default = {engine = rocksdb, size = 16777216, writeBufferSize = 67108864}")
```

Comment:
Please indicate in the comment why you set this value, for example: 
The writeBufferSize is large enough to reduce the flush interval and speed up tests.

Reply:
Done